features/comments_and_kudos/add_comment.feature

```diff
@@ -27,8 +29,24 @@ Scenario: When logged in I can comment on a work
     And I follow "Entire Work"
     And I follow "Comments (1)"
   Then I should see "commenter on Chapter 1" within "h4.heading.byline"
-    
+
+Scenario: Comment editing
+
+  When I am logged in as "author"
+    And I post the work "The One Where Neal is Awesome"
+  When I am logged in as "commenter"
+    And I post the comment "Mistaken comment" on the work "The One Where Neal is Awesome"
+    And I follow "Edit"
+  And I fill in "Comment" with "Actually, I meant something different"
+    And I press "Update"
+  Then I should see "Comment was successfully updated"
+    And I should see "Actually, I meant something different"
+    And I should not see "Mistaken comment"
+    And I should see "EST" within ".comment .posted"
```

Comment:
I am confused / worried about hardcoding this timezone here. Since @cesy is travelling, if anyone else can explain if/why it's safe to do so that would also be good!